CHANGELOG.md

```diff
@@ -40,6 +40,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Removed
+- **Breaking**: Removes deprecated `org.partiql.annotations.PartiQLExperimental`
+- **Breaking**: Removes deprecated constructor for `org.partiql.lang.CompilerPipeline`
+- **Breaking**: Removes deprecated `valueFactory`, `build`, `builder`, `standard` in `org.partiql.lang.CompilerPipeline`
+- **Breaking**: Removes deprecated constructor in `org.partiql.lang.CompilerPipeline`
+- **Breaking**: Removes deprecated `ofIonStruct()` from `org.partiql.lang.eval.Bindings`
+- **Breaking**: Removes deprecated/unused `blacklist()` and `denyList()` from `org.partiql.lang.eval`
+- **Breaking**: Removes deprecated enum `LEGACY` in `org.partiql.lang.eval.CompileOptions`
+- **Breaking**: Removes deprecated `org.partiql.lang.eval.ExprValueFactory`
+- **Breaking**: Removes deprecated `isNull()` in `org.partiql.lang.eval.ExprValueType`
+- **Breaking**: Removes deprecated `convert()` in `org.partiql.lang.eval.io.DelimitedValues.ConversionMode`
+- **Breaking**: Removes deprecated `exprValue()` from `org.partiql.lang.eval.io.DelimitedValues`
+- **Breaking**: Removes deprecated constructor for `org.partiql.lang.eval.physical.EvaluatorState`
+- **Breaking**: Removes deprecated `org.partiql.lang.eval.visitors.GroupKeyReferencesVisitorTransform`
```

Comment:
This list of **Breaking** removals may sound more alarming than necessary.  In many cases, a method that has an `ExprValueFactory` argument is being removed, while its counterpart without such an argument is still there. One example is `Bindings.ofIonStruct`, but  there are more. 

My preference would be to spell this out in CHANGELOG to preclude any panic, while reducing the repetitiveness as well.  Something like: 

> **Breaking**: Removes deprecated `org.partiql.lang.eval.ExprValueFactory`, as well as all methods (previously deprecated as well) that had its instance among arguments, namely: `ofIonStruct()` in `org.partiql.lang.eval.Bindings`, ...[TODO]....  The counterparts of these methods without an `ExprValueFactory` are still available.